server/routes/journeys/update-plan/check-answers/view.njk

```diff
@@ -0,0 +1,124 @@
+            },
+            actions: {
+              items: [
+                {
+                  href: "intervention-details#responsiblePerson",
+                  text: "Change",
+                  visuallyHiddenText: "the person responsible",
+                  classes: "govuk-link--no-visited-state"
+                }
+              ]
+            }
+          },
+          {
+            key: {
+                text: 'Target date'
+            },
+            value: {
+                text: need.targetDate | formatDisplayDate
+            },
+            actions: {
+              items: [
+                {
+                  href: "intervention-details#targetDate",
+                  text: "Change",
+                  visuallyHiddenText: "the target date",
+                  classes: "govuk-link--no-visited-state"
+                }
+              ]
+            }
+          },
+          {
+            key: {
+                text: 'Identified need summary'
+            },
+            value: {
+                html: (need.identifiedNeed or 'Not provided') | escape | boldAppendStamp | nl2br
+            },
+            actions: {
+              items: [
+                {
+                  href: "summarise-identified-need#identifiedNeed",
+                  text: "Change",
+                  visuallyHiddenText: "the summary of the identified need",
+                  classes: "govuk-link--no-visited-state"
+                }
+              ]
+            } if not need.closedDate else undefined
+          },
+          {
+            key: {
+                text: 'Planned intervention'
+            },
+            value: {
+                html: (need.intervention or 'Not provided') | escape | boldAppendStamp | nl2br
+            },
+            actions: {
+              items: [
+                {
+                  href: "intervention-details#intervention",
+                  text: "Add information",
+                  visuallyHiddenText: "to the planned intervention",
+                  classes: "govuk-link--no-visited-state"
+                }
+              ]
+            } if not need.closedDate else undefined
+          },
+          {
+            key: {
+                text: 'Actions and progress'
+            },
+            value: {
+                html: (need.progression or 'Not provided') | escape | boldAppendStamp | nl2br
+            },
+            actions: {
+              items: [
+                {
+                  href: "record-actions-progress#progression",
+                  text: "Add information",
+                  visuallyHiddenText: "to the actions and progress",
+                  classes: "govuk-link--no-visited-state"
+                }
+              ]
+            } if not need.closedDate else undefined
+          }]
+      }) }}
+
+    <form method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+        <div class="govuk-button-group">
```

Comment:
`submitButton` already has  govuk-button-group built in. we don't need this div here